plugins/pay.c

```diff
@@ -262,7 +262,7 @@ static struct command_result *waitsendpay_expired(struct command *cmd,
 	for (size_t i = 0; i < tal_count(pc->ps->attempts); i++) {
 		json_object_start(data, NULL);
 		if (pc->ps->attempts[i].route)
-			json_add_member(data, "route", false, "%s",
+			json_add_member(data, "route", true, "%s",
```

Comment:
Hmm, this looks like json_add_string() would be better?  add_member is pretty low-level...

Reply:
+1, but fixed in https://github.com/ElementsProject/lightning/pull/3632